config.js

```diff
@@ -3,5 +3,10 @@
 module.exports = {
   "paypal": {
     "hosted_button_id": "ETEEHS7H7NBFJ"
+  },
+  "facebook": {
```

Comment:
These values correspond to a test app I have and are valid to try this new feature in localhost. You should go to http://developers.facebook.com/ and create a new app for Books4Equality.